maintenance_equipment_category_hierarchy/readme/CONTRIBUTORS.md

```diff
@@ -0,0 +1,6 @@
+- Enric Tobella
+- Jaime Arroyo
+- Olga Marco
+- Kevin Luna
+- [Heliconia Solutions Pvt. Ltd.](<https://www.heliconia.io>)
+  - Bhavesh Heliconia
```

Comment:
For basic migration adding yourself to contributors shouldn't be accepted, unless a new feature is provided IMO